shared/src/commonMain/kotlin/co/touchlab/kampkit/Koin.kt

```diff
@@ -3,19 +3,26 @@ package co.touchlab.kampkit
 import co.touchlab.kampkit.ktor.DogApiImpl
 import co.touchlab.kampkit.ktor.KtorApi
 import kotlinx.coroutines.Dispatchers
+import org.koin.core.KoinApplication
 import org.koin.core.context.startKoin
 import org.koin.core.module.Module
 import org.koin.core.parameter.parametersOf
 import org.koin.core.scope.Scope
-import org.koin.dsl.KoinAppDeclaration
 import org.koin.dsl.module
 
-fun initKoin(appDeclaration: KoinAppDeclaration = {}) = startKoin {
-    appDeclaration()
-    modules(
-        platformModule,
-        coreModule
-    )
+fun initKoin(appModule: Module): KoinApplication {
+    val koinApplication = startKoin {
```

Comment:
I'm not sure we want to pass in a module instead of a KoinAppDeclaration. There are other configurations (logger, properties) that it might be worthwhile for the platform to have access to

Reply:
If I understand right, the platform should still be able to pull out whatever else it wants. This just simplifies the setup syntax since the platforms were just passing a single module anyway.

Yes this implies just a single module which you may not want if you have a large amount of dependencies, but for the scale of KaMPKit I don't see that as an issue. Once you've done enough refactoring that you've added lots of app-level dependencies it's a very low lift to also refactor this structure.